docs/tools/vdb_table/data/pinecone.json

```diff
@@ -68,38 +68,38 @@
     "comment": ""
   },
   "embeddings_text": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/pinecone-io/canopy",
     "comment": ""
   },
   "embeddings_image": {
-    "support": "none",
-    "source_url": "",
+    "support": "",
```

Comment:
Keeping it as "none" would be correct here.

Reply:
added